spec/serializers/person_escort_record_serializer_spec.rb

```diff
@@ -137,4 +137,41 @@
       expect(result[:included]).to include_json(expected_json)
     end
   end
+
+  describe 'generic_events' do
+    let(:includes) { %i[events] }
+
+    context 'with generic events' do
+      let(:now) { Time.zone.now }
+      let!(:first_event) { create(:event_per_court_hearing, eventable: person_escort_record, occurred_at: now + 2.seconds) }
+      let!(:second_event) { create(:event_per_court_cell_share_risk_assessment, eventable: person_escort_record, occurred_at: now + 1.second) }
+      let!(:third_event) { create(:event_per_court_all_documentation_provided_to_supplier, eventable: person_escort_record, occurred_at: now) }
+
+      let(:expected_event_relationships) do
+        [
+          { id: third_event.id, type: 'events' },
+          { id: second_event.id, type: 'events' },
+          { id: first_event.id, type: 'events' },
+        ]
+      end
+
+      it 'contains event relationships' do
```

Comment:
could you add something in the test description to indicate that order matters here? (Same everywhere)

Reply:
Sure